pageserver/compaction/src/simulator/draw.rs

```diff
@@ -160,9 +160,12 @@ pub fn draw_history<W: std::io::Write>(history: &[LayerTraceEvent], mut output:
 
         // Fill in and thicken rectangle if it's an
         // image layer so that we can see it.
-        let mut style = Style::default();
-        style.fill = Fill::Color(rgb(0x80, 0x80, 0x80));
-        style.stroke = Stroke::Color(rgb(0, 0, 0), 0.5);
+        let mut style = Style {
```

Comment:
This seems unrelated to the PR?

(I think this is what triggered storage review request)

Reply:
It was related to the axum upgrade work. Clippy was saying this was bad for some reason. I will be rebasing this PR, so the diff should minimize here.